test/ModuleInterface/has_storage_attr_bad.swiftinterface

```diff
@@ -1,9 +0,0 @@
-// swift-interface-format-version: 1.0
```

Comment:
Please restore this file even if it no longer has an expected-warning line; it also is a "this input doesn't crash" test.

Reply:
Done